torch/torch.go

```diff
@@ -0,0 +1,20 @@
+package torch
+
+// #cgo CFLAGS: -I ${SRCDIR}/../cgotorch
+// #cgo LDFLAGS: -L ${SRCDIR}/../cgotorch -Wl,-rpath ${SRCDIR}/../cgotorch -lcgotorch
+// #cgo LDFLAGS: -L ${SRCDIR}/../cgotorch/libtorch/lib -Wl,-rpath ${SRCDIR}/../cgotorch/libtorch/lib -lc10 -ltorch -ltorch_cpu
+// #include "cgotorch.h"
+import "C"
+import (
+	"github.com/wangkuiyi/gotorch/aten"
+	"unsafe"
+)
+
+// RandN returns a tensor filled with random number
+func RandN(rows, cols int, requireGrad bool) aten.Tensor {
+	rg := 0
+	if requireGrad {
+		rg = 1
+	}
+	return aten.NewTensor(unsafe.Pointer(C.RandN(C.int(rows), C.int(cols), C.int(rg))))
```

Comment:
So, it's illegal to write code like `aten.Tensor{C.RandN(C.int(rows), C.int(cols), C.int(rg))}`?

Reply:
Yes, it raied an error as the following, Go compiler considers the C types come from different packages are the different types.


``` text
torch/torch.go:21:28: cannot use _Cfunc_RandN(_Ctype_int(rows), _Ctype_int(cols), _Ctype_int(rg)) (type _Ctype_Tensor) as type aten._Ctype_Tensor in field value
```